Extension/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.13.5: November 28, 2022
+### Bug Fixes
+* Fix "Step Over past a logpoint stops at the wrong place". [#9995](https://github.com/microsoft/vscode-cpptools/issues/9995)
+* Fix "Don't hardcode path to kill in UnixUtilities". [#10124](https://github.com/microsoft/vscode-cpptools/issues/10124)
+	* Thank you by @expipiplus1
```

Comment:
This looks over indented.  The word "by" seems unnecessary.  And I don't think we normally say an explicit "Thank you" when we do attributions.

Reply:
Yeah, I just copied that from what Andrew wrote -- I'll modify the style to match our existing attribution approach.